src/feedback/views/intern/vv.py

```diff
@@ -150,7 +150,10 @@ def has_similar_name(self):
         nachname = self.object.nachname
         similar_persons = Person.persons_with_similar_names(vorname, nachname)
 
-        return similar_persons.count() > 0
+        if self.object.full_name() == similar_persons.get().full_name():
```

Comment:
I guess this code throws an DoesNotExist exception if there is no similar person. So use ```.exists()``` to check if the queryset is not empty and to compare two django objects better use
```python
self.object == similar_persons.get()
```